test/test_fx.py

```diff
@@ -1597,6 +1597,42 @@ def test_no_mutation(self):
         with self.assertRaisesRegex(NotImplementedError, "new_args"):
             x[0] = 4
 
+    def test_error_reporting_after_recompile(self):
+        try:
+            from io import StringIO
+            from unittest.mock import patch
+        except ImportError:
+            self.skipTest("Necessary modules not available")
+
+        class M(torch.nn.Module):
+            def __init__(self):
+                super(M, self).__init__()
+                self.W = torch.nn.Parameter(torch.randn(5))
+
+            def forward(self, x):
+                return torch.dot(self.W, x)
+
+        traced = torch.fx.symbolic_trace(M())
+
+        out = [n for n in list(traced.graph.nodes) if n.op == "output"]
```

Comment:
nit: Don't think you need to put all the nodes in a list for this for-comp to work